src/content/docs/data-apis/ingest-apis/event-api/incident-event-rest-api.mdx

```diff
@@ -0,0 +1,312 @@
+              "triggeredAt": 1641909163980,
+              "entitiesData.ids": "testEntity123",
+              "entitiesData.types": "unknown",
+              "labelsHash": "91f938318e899dcd62965e2724548f4844f2898f3cf1c5411397ecc0eec87fc1",
+              "annotations.description": "[\"The latency is above threshold of 500000 MS\"]",
+              "labels.originalAccountIds": "<var>YOUR_ACCOUNT_ID</var>",
+              "entitiesData.names": "testEntity",
+              "entitiesData.entities": "{\"id\":\"testEntity123\",\"name\":\"testEntity\",\"type\":\"unknown\"}",
+              "priority": "HIGH",
+              "labels.serviceId": "5",
+              "labels.environment": "testing",
+              "closedAt": "",
+              "updatedAt": 1641910123824,
+              "annotations.title": "[\"Test\",\"Test2222\",\"Test3\"]",
+              "nrAccountId": <var>YOUR_ACCOUNT_ID</var>,
+              "accumulations": "{\"source\":[\"rest\"],\"origin\":[\"luna\"],\"entity_guid\":[\"testEntity123\"],\"tag.stackTrace\":[\"some stack trace...\"]}",
+              "labels.accountIds": "<var>YOUR_ACCOUNT_ID</var>",
+              "createdAt": 1641909163997,
+              "priority.order": 2,
+              "labels.aggregationKeys": "416e2e55069764086ad83e97a1160be0cb4c545b7950522f7c86baaa2a11b9b6",
+              "accumulations.origins": "[\"luna\"]",
+              "accumulations.entity_guid": "[\"testEntity123\"]",
+              "incidentId": "a06ffb92-2f7e-473b-953f-151ff4777cb0",
+              "labels.errorId": "10543",
+              "dataMLModules": "{}",
+              "triggerEvent": "VIOLATION_ADDED",
+              "accumulations.tag.stackTrace": "[\"some stack trace...\"]",
+              "isIint": false,
+              "accumulations.sources": "[\"rest\"]",
+              "entitiesData": "{\"name\":\"testEntity\",\"id\":\"testEntity123\",\"type\":\"unknown\",\"entities\":\"{\\\"id\\\":\\\"testEntity123\\\",\\\"name\\\":\\\"testEntity\\\",\\\"type\\\":\\\"unknown\\\"}\"}",
+              "annotations": "{\"title\":[\"Test\",\"Test2222\",\"Test3\"],\"description\":[\"The latency is above threshold of 500000 MS\"]}",
+              "labels": "{\"accountId\":\"XXXXXXX\",\"originalAccountId\":\"XXXXXXX\",\"entityId\":\"testEntity123\",\"entityName\":\"testEntity\",\"entityType\":\"unknown\",\"aggregationKey\":\"416e2e55069764086ad83e97a1160be0cb4c545b7950522f7c86baaa2a11b9b6\",\"serviceId\":\"5\",\"errorId\":\"10543\",\"environment\":\"testing\"}",
+              "state": "CREATED"
+            }
+```
+
+## API specifications [#api-specs]
+
+For descriptions of our value types, see [our event API JSON guidelines](/docs/data-apis/ingest-apis/introduction-event-api/#json-guidelines).
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Field
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
```

Comment:
not sure if this causes build failures, but i _think_ we may want a new, empty line between `thead` and `tbody` here

Reply:
This is the first I've heard of this. This is the standard table markup that we have in keyboard maestro. Maybe it'd be good to write up a list of little gotchas like this?